include/mirage/utils/fingerprint_functions.h

```diff
@@ -81,15 +81,36 @@ inline __device__ FPType compute_silu_fingerprint(FPType input,
 }
 
 inline __device__ FPType compute_gelu_fingerprint(FPType input,
-  FPType *exp_lookup_table) {
-// Approximating GeLU as x*sigmoid(1.702x)
-FPType q_residual = input % FP_Q;
-FPType scaling_factor = 1.702f;
-FPType scaled_q_residual = (scaling_factor * q_residual) % FP_Q;
+                                                  FPType *exp_lookup_table) {
+  // Approximating GeLU as x*sigmoid(1.702x)
+  FPType q_residual = input % FP_Q;
+  FPType scaling_factor = 1.702f;
```

Comment:
This would not work due to type casting? IIRC it was introduced in a previous PR, so we can either fix it now or in a separate PR

Reply:
Let's fix it in a separate PR.